react/Modal/index.jsx

```diff
@@ -82,5 +83,5 @@ Modal.defaultProps = {
   withCross: true
 }
 
-export { styles }
+export { Content }
```

Comment:
If I understand well, the actual goal is to avoid a double import and write something like :
`import Modal, { Content as ModalContent } from 'cozy-ui/react/Modal'` ? I like it.